streaming/src/test/scala/org/apache/spark/streaming/util/WriteAheadLogSuite.scala

```diff
@@ -366,6 +367,21 @@ object WriteAheadLogSuite {
     segments
   }
 
+  /**
+   * Write received block tracker events to a file using the writer class and return an array of
+   * the file segments written.
+   */
+  def writeEventsUsingWriter(
```

Comment:
This method is related to `ReceivedBlockTracker` and should not be in this file, as this file is totally independent. Move this method to `ReceivedBlockTrackerSuite`.